content/guides/accounts-synchronization.md

```diff
@@ -0,0 +1,184 @@
+
+> Note: Pagination is not supported for this endpoint.
+
+> Important: If an account is no longer visible in the list, ensure to disable the corresponding account and rentals.
+
+> Recommendation: We suggest to do synchronization of accounts every 12-24 hours.
+
+
+## Code example in cURL
+
+~~~bash
+TOKEN="<YOUR_TOKEN>"
+API_URL="<API_URL>"
+
+curl -X GET \
+  "$API_URL/api/ota/v1/accounts" \
+  -H "User-Agent: API Client" \
+  -H "Accept: application/vnd.api+json" \
+  -H "Content-Type: application/vnd.api+json" \
+  -H "Authorization: Bearer $TOKEN"
+~~~
+
+## Code example in Ruby
+
+~~~ruby
+require 'excon'
+require 'json'
+
+token = "<YOUR_TOKEN>"
+api_url = "<API_URL>"
+media_type = "application/vnd.api+json"
+options = {
+  headers: {
+    "User-Agent" => "Api client",
+    "Accept" => media_type,
+    "Content-Type" => media_type,
+    "Authorization" => "Bearer #{token}"
+  }
+}
+request = Excon.new(URI.join(api_url, "/api/ota/v1/accounts").to_s, options)
+response = request.request(method: :get)
+
+response.status
+
+already_import_accounts_ids = get_imported_accounts_ids # get an array of already imported accounts, Ex result: [1,2,3]
+
+json = JSON.parse(response.body)
+json["data"].each do |account|
+  # Import accounts
+  import_account(account["id"], account["attributes"]["name"])
+end
+
+accounts_for_disabling = already_import_accounts_ids - json["data"].pluck("id")
+disable_accounts_and_rentals(accounts_for_disabling) # Disable accounts
+~~~
+
+Replace `<YOUR_TOKEN>` and `<API_URL>` with your specific authentication token and API URL in the provided example.
+
+## Code example in Python
+
+~~~python
+import requests
+
+token = "<YOUR_TOKEN>"
+api_url = "<API_URL>"
+media_type = "application/vnd.api+json"
+headers = {
+    "User-Agent": "API Client",
+    "Accept": media_type,
+    "Content-Type": media_type,
+    "Authorization": f"Bearer {token}"
+}
+response = requests.get(f"{api_url}/api/ota/v1/accounts", headers=headers)
+
+response_status = response.status_code
+
+# Assuming you have a function to retrieve already imported account IDs: get_imported_accounts_ids
+already_import_accounts_ids = get_imported_accounts_ids()
+
+json_data = response.json()
+for account in json_data["data"]:
+    # Import accounts
+    import_account(account["id"], account["attributes"]["name"])
+
+accounts_for_disabling = [account_id for account_id in already_import_accounts_ids if account_id not in [a["id"] for a in json_data["data"]]]
+disable_accounts_and_rentals(accounts_for_disabling)  # Disable accounts
+~~~
+
+Please replace `<YOUR_TOKEN>` and `<API_URL>` with your actual authentication token and API URL in the Python code. Additionally, make sure you have the necessary functions **get_imported_accounts_ids**, **import_account**, and **disable_accounts_and_rentals** defined and implemented in your Python script.
```

Comment:
since we also have JAVA example I think worth to mention both variants - **get_imported_accounts_ids/getImportedAccountsIds**, **import_account/importAccount**, and **disable_accounts_and_rentals/disableAccountsAndRentals**